charts/neuvector/100.0.1+up2.2.2/Chart.yaml

```
@@ -0,0 +1,28 @@
+annotations:
+  catalog.cattle.io/auto-install: neuvector-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: NeuVector
+  catalog.cattle.io/kube-version: '>=1.18.0-0'
```

Was this intentionally left unbounded?
```suggestion
  catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.25.0-0'
```

---

I will make changes and resubmit.